Automattic/wp-calypso
# Marketplace: Redirect to vendor return url after the checkout

#### Proposed Changes

* In D91398-code we added an additional field for redirecting users to the vendor app upon successful checkout. This PR looks for the `saas_redirect_url` field in the purchases array and redirects users to that URL if the field is not empty. The field `saas_redirect_url` only be set from the backend when the product is a SaaS product. 

#### Testing Instructions



* Apply diff D91398-code on your sandbox if not already deployed. 
* Follow test instructions provided in D91398-code. 
* Make sure you are redirected to the redirect URL after successful checkout when following test instructions from the diff. 

#### Pre-merge Checklist






Related to #68148
